platform/binance/api_test.go

```diff
@@ -85,107 +83,105 @@ const cancelOrderTransaction = `
-		Decimals: 8,
-	},
-}
-
-//TODO: temp test dst
-var metaFreeCancelOrdeTransferDst = blockatlas.Tx{
-	ID:     "F48DE755170C10F4A4C0E6836A708C33EEF9A7144800F25187D5F2349FD15A34",
-	Coin:   coin.BNB,
-	From:   "bnb16ya67j7kvw8682kka09qujlw5u7lf4geqef0ku",
-	Fee:    "0",
-	Date:   1559689892,
-	Block:  10815539,
-	Status: blockatlas.StatusCompleted,
-	Meta:   nil,
-}
+var (
+	transferDst = blockatlas.Tx{
+		ID:     "1681EE543FB4B5A628EF21D746E031F018E226D127044A4F9BA5EE2542A44555",
+		Coin:   coin.BNB,
+		From:   "tbnb1fhr04azuhcj0dulm7ka40y0cqjlafwae9k9gk2",
+		To:     "tbnb1sylyjw032eajr9cyllp26n04300qzzre38qyv5",
+		Fee:    "125000",
+		Date:   1555049867,
+		Block:  7761368,
+		Status: blockatlas.StatusCompleted,
+		Memo:   "test",
+		Meta: blockatlas.Transfer{
+			Value:    "10000000000000",
+			Decimals: 8,
+			Symbol:   "BNB",
+		},
+	}
+	tokenTransferDst = blockatlas.Tx{
+		ID:     "95CF63FAA27579A9B6AF84EF8B2DFEAC29627479E9C98E7F5AE4535E213FA4C9",
+		Coin:   coin.BNB,
+		From:   "tbnb1ttyn4csghfgyxreu7lmdu3lcplhqhxtzced45a",
+		To:     "tbnb12hlquylu78cjylk5zshxpdj6hf3t0tahwjt3ex",
+		Fee:    "125000",
+		Date:   1555117625,
+		Block:  7928667,
+		Status: blockatlas.StatusCompleted,
+		Memo:   "test",
+		Meta: blockatlas.NativeTokenTransfer{
+			TokenID:  "YLC-D8B",
+			Symbol:   "YLC",
+			Value:    "210572645",
+			Decimals: 8,
+			From:     "tbnb1ttyn4csghfgyxreu7lmdu3lcplhqhxtzced45a",
+			To:       "tbnb12hlquylu78cjylk5zshxpdj6hf3t0tahwjt3ex",
+		},
+	}
+	newOrderTransferDst = blockatlas.Tx{
+		ID:     "B0677F3436C1B1661E94D192B84B98AA42AC2485D9808357796EE501CBF794F7",
+		Coin:   coin.BNB,
+		From:   "bnb16ya67j7kvw8682kka09qujlw5u7lf4geqef0ku",
+		Fee:    "0",
+		Date:   1559689901,
+		Block:  10815565,
+		Status: blockatlas.StatusCompleted,
+		Meta: blockatlas.AnyAction{
+			Coin:     coin.BNB,
+			Title:    blockatlas.KeyTitlePlaceOrder,
+			Key:      blockatlas.KeyPlaceOrder,
+			TokenID:  "BNB_AWC-986",
+			Name:     "AWC-986",
+			Symbol:   "AWC",
+			Value:    "649878",
+			Decimals: 8,
+		},
+	}
+	//TODO: temp test dst
+	metaFreeNewOrderTransferDst = blockatlas.Tx{
+		ID:     "B0677F3436C1B1661E94D192B84B98AA42AC2485D9808357796EE501CBF794F7",
+		Coin:   coin.BNB,
+		From:   "bnb16ya67j7kvw8682kka09qujlw5u7lf4geqef0ku",
+		Fee:    "0",
+		Date:   1559689901,
+		Block:  10815565,
+		Status: blockatlas.StatusCompleted,
+		Meta:   nil,
+	}
+	cancelOrdeTransferDst = blockatlas.Tx{
+		ID:     "F48DE755170C10F4A4C0E6836A708C33EEF9A7144800F25187D5F2349FD15A34",
+		Coin:   coin.BNB,
+		From:   "bnb16ya67j7kvw8682kka09qujlw5u7lf4geqef0ku",
+		Fee:    "0",
+		Date:   1559689892,
+		Block:  10815539,
+		Status: blockatlas.StatusCompleted,
+		Meta: blockatlas.AnyAction{
+			Coin:     coin.BNB,
+			Title:    blockatlas.KeyTitleCancelOrder,
+			Key:      blockatlas.KeyCancelOrder,
+			TokenID:  "BNB_GTO-908",
+			Name:     "GTO-908",
+			Symbol:   "GTO",
+			Value:    "104716",
+			Decimals: 8,
+		},
+	}
+	//TODO: temp test dst
```

Comment:
can you give more details?

Reply:
https://golang.org/doc/effective_go.html#constants

https://golang.org/doc/effective_go.html#variables

Reply:
What is this TODO for?

Reply:
I have no idea, these todos were before my commits. I can remove them
https://github.com/trustwallet/blockatlas/blob/master/platform/binance/api_test.go#L146
https://github.com/trustwallet/blockatlas/blob/master/platform/binance/api_test.go#L178